funfact/lang/interpreter/_initialization.py

```diff
@@ -30,6 +30,8 @@ def scalar(self, value, **kwargs):
     @as_payload
     def tensor(self, value, **kwargs):
         if value.initializer is not None:
+            if not callable(value.initializer):
+                return value.initializer
```

Comment:
We'll probably need to return a copy of the initializer if it is a tensor so that the original value would not be changed by any following operations.